packages/expandable-card/src/ExpandableCard.story.tsx

```diff
@@ -30,6 +30,7 @@ storiesOf('ExpandableCard', module)
         )}
         isOpen={isOpen}
         onClick={e => {
+          // eslint-disable-next-line no-console
```

Comment:
Is it possible to have separate lint rules for `.story` files?

Reply:
I looked, and came up short on that unfortunately. Went with the comments unless we find something.